README.md

```diff
@@ -1 +1,202 @@
-# terraform-prefect-ecs-worker
+# Terraform Prefect ECS Module
+
+This module deploys a Prefect 3 worker onto ECS Fargate using
```

Comment:
I don't think we need to specify `Prefect 3` here

Reply:
True, updated in e82bb09